src/main/java/com/googleinterns/zoomtube/servlets/TranscriptServlet.java

```diff
@@ -41,22 +55,27 @@
   private static final String START_ATTRIBUTE = "start";
   private static final String DURATION_ATTRIBUTE = "dur";
   private static final String TEXT_TAG = "text";
-  // TODO: Use actual video id.
-  private static final String TEST_VIDEO_ID = "3ymwOvzhwHs";
+  private static final String PARAM_LECTURE = "lecture";
+  private static final String PARAM_LECTURE_ID = "id";
+  private static final String PARAM_VIDEO_ID = "video";
+
+  private static DatastoreService datastore;
 
   @Override
   public void init() throws ServletException {
-    // TODO: Implement Transcript.
+    this.datastore = DatastoreServiceFactory.getDatastoreService();
   }
 
   @Override
-  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    // TODO: Pass the video ID from another servlet.
-    String transcriptXMLUrl = TRANSCRIPT_XML_URL_TEMPLATE + TEST_VIDEO_ID;
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
```

Comment:
See comment I left on `doPost()`, consider decomposing

Reply:
commented below, will fix in later pull request (#59).